ostd/src/cpu/mod.rs

```diff
@@ -114,6 +114,11 @@ impl CpuSet {
         self.bitset.get(cpu_id as usize).as_deref() == Some(&true)
     }
 
+    /// Returns the number of CPUs in the set.
+    pub fn nr_cpus(&self) -> usize {
```

Comment:
I think calling this method `count` is better. This name is more consistent with the names of other methods, which do not include `cpu`. And it gives a hint to its caller that this method is not free (as you have to count something).